src/testing/jest/jest-preprocessor.ts

```diff
@@ -14,7 +14,7 @@ type Jest27TransformOptions = { config: Jest26Config };
  * @returns `true` if the `obj` is a `Jest27TransformOptions`, false otherwise
  */
 const isJest27TransformOptions = (obj: unknown): obj is Jest27TransformOptions => {
-  return typeof obj === 'object' && obj.hasOwnProperty('config');
+  return obj != null && typeof obj === 'object' && obj.hasOwnProperty('config');
```

Comment:
Because `typeof obj === 'object'` is true when `obj` is `null` 🤦

Reply:
just a lovely little bit of JS lore 😂